attributes/default.rb

```diff
@@ -66,7 +66,18 @@
       '/media/downloadable/',
       '/pkginfo/',
       '/report/config.xml',
-      '/var/'
+      '/var/',
+      '/.gitignore',
+      '/.htaccess',
+      '/.htaccess.sample',
+      '/composer.json',
+      '/cron.php',
+      '/cron.sh',
+      '/index.php.sample',
+      '/install.php',
+      '/mage',
```

Comment:
there's no mage in the magento repository. If this is just for your project you can put it in your project's project-web.json